devices/generic/subdevices/extended_color_light.json

```diff
@@ -6,9 +6,15 @@
     "order": 10,
     "uuid": ["$address.ext", "0x01"],
     "items": [
+        "cap/color/capabilities",
+        "state/alert",
+        "state/colormode",
+        "state/on",
+        "state/reachable"
+    ],
+    "items_optional":[
         "cap/alert/trigger_effect",
         "cap/bri/move_with_onoff",
-        "cap/color/capabilities",
         "cap/color/ct/max",
```

Comment:
`cap/color/ct/max` and `cap/color/ct/min` are mandatory, as _Extended Color Light_ also has mandatory `state/ct`.